README.rst

```diff
@@ -2,6 +2,8 @@
 Automatically updated list of valid TLDs for Python
 ###################################################
 
+*Fork that does not require internet access during build.*
```

Comment:
We should probably remove this 😄

Reply:
Indeed. Happy to upstream those changes!